packages/elasticsearch-client/src/config/default/resources/index-document.ts

```diff
@@ -64,7 +74,8 @@ export const convertDocument = (resource: ReferrableResource): ResourcesIndexDoc
 
   return {
     name,
-    text1,
-    text2,
+    high_boost_global: highBoostGlobal.join(' '),
```

Comment:
I moved from arrays to a more common giant text string search which returns more reliable scoring/ordering, in my opinion. You'll see below that the results in tests in resources-service are ordered by documents that contain repeat values more often, which I think is likely the more desirable behavior.